src/MultiDropdown.js

```diff
@@ -3,19 +3,79 @@ import PropTypes from "prop-types";
 import * as countriesCities from "countries-cities";
 import "./MultiDropdown.css";
 import _ from "lodash";
+import CustomDropdown from "./CustomDropdown";
 
 // HINT User CustomDropdown.js here
 
 export default class MultiDropdown extends Component {
   constructor(props) {
     super(props);
-    this.state = {};
+    this.state = {
+      country: null,
+      cities: null
+    };
+  }
+  selectCountries(country) {
```

Comment:
you're selecting one country, not multiple countries. This is a bad name.

Reply:
done